Funcky/Extensions/ParseExtensions/ParseExtensions.GenericParseable.cs

```diff
@@ -0,0 +1,12 @@
+#if GENERIC_PARSEABLE
+namespace Funcky.Extensions;
+
+public static partial class ParseExtensions
+{
+    public static Option<TParseable> ParseGenericParsable<TParseable>(this ReadOnlySpan<char> value, IFormatProvider? provider)
```

Comment:
There's also the `IParseable` interface...

Reply:
Added in: c4d790f687551bfda36eb15615b06ba23ee39656